docs/index.md

```diff
@@ -36,27 +36,18 @@ These topics are the most visited ones, we thought you’d like to have a glance
     <a class="tp-content-card" href="manuals/reference">
       <img class="tp-content-card-icon icon-light" src="images/icons/menu_book-w.svg">
       <img class="tp-content-card-icon icon-dark" src="images/icons/menu_book.svg">
-      <h3>Reference Manual</h3>
+      <h3>User Manual</h3>
```

Comment:
Either the title or the description is incorrect. 
The documentation for all APIs exposed in Taipy is the Reference manual.
The user man is more a functional documentation with standard examples.

Reply:
It has been asked from me just to replace the title... I wasn't sure it had been discussed with you guys from R&D beforehand, but I was told it did.
I'll check if I misunderstood the demand or else...

Reply:
ok, thx.